help/Makefile.am

```diff
@@ -1,16 +1,16 @@
-include $(top_srcdir)/gnome-doc-utils.make
-dist-hook: doc-dist-hook
+@YELP_HELP_RULES@
```

Comment:
What does this line do exactly? AFAICS we're not actually executing the Makefile, only parsing it to get some variables? Is this needed then? Or does it allow manually building with make?

Reply:
It's not really needed. It's just needed if someone wants to use autotools.

Reply:
I wonder if that is still relevant, but removing this is probably out of scope for this PR, so good to keep it working for now.